From 67979163ac3abee8415f8b178f45e9a111d9c6e5 Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Wed, 10 Aug 2016 21:02:01 +0800
Subject: [PATCH] arm64: dts: rockchip: modify dwc3 properties for rk3399

We have merged dwc3 driver from upstream, and some properties
need to be modified according to upstream coding style.

Change-Id: I4f8c4b23a941932a08eb29a0282dfb0903193c8a
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 31f4ca50c219..709970d7191a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -431,7 +431,7 @@
 	};
 
 	usbdrd3_0: usb@fe800000 {
-		compatible = "rockchip,dwc3";
+		compatible = "rockchip,rk3399-dwc3";
 		clocks = <&cru SCLK_USB3OTG0_REF>, <&cru SCLK_USB3OTG0_SUSPEND>,
 			 <&cru ACLK_USB3OTG0>, <&cru ACLK_USB3_RKSOC_AXI_PERF>,
 			 <&cru ACLK_USB3>, <&cru ACLK_USB3_GRF>;
@@ -450,17 +450,17 @@
 			dr_mode = "otg";
 			phys = <&u2phy0_otg>;
 			phy-names = "usb2-phy";
+			phy_type = "utmi_wide";
 			snps,dis_enblslpm_quirk;
-			snps,phyif_utmi_16_bits;
-			snps,dis_u2_freeclk_exists_quirk;
-			snps,dis_del_phy_power_chg_quirk;
-			snps,xhci_slow_suspend_quirk;
+			snps,dis-u2-freeclk-exists-quirk;
+			snps,dis-del-phy-power-chg-quirk;
+			snps,xhci-slow-suspend-quirk;
 			status = "disabled";
 		};
 	};
 
 	usbdrd3_1: usb@fe900000 {
-		compatible = "rockchip,dwc3";
+		compatible = "rockchip,rk3399-dwc3";
 		clocks = <&cru SCLK_USB3OTG1_REF>, <&cru SCLK_USB3OTG1_SUSPEND>,
 			 <&cru ACLK_USB3OTG1>, <&cru ACLK_USB3_RKSOC_AXI_PERF>,
 			 <&cru ACLK_USB3>, <&cru ACLK_USB3_GRF>;
@@ -479,11 +479,11 @@
 			dr_mode = "otg";
 			phys = <&u2phy1_otg>;
 			phy-names = "usb2-phy";
+			phy_type = "utmi_wide";
 			snps,dis_enblslpm_quirk;
-			snps,phyif_utmi_16_bits;
-			snps,dis_u2_freeclk_exists_quirk;
-			snps,dis_del_phy_power_chg_quirk;
-			snps,xhci_slow_suspend_quirk;
+			snps,dis-u2-freeclk-exists-quirk;
+			snps,dis-del-phy-power-chg-quirk;
+			snps,xhci-slow-suspend-quirk;
 			status = "disabled";
 		};
 	};
-- 
2.35.3

